src/codeenginestudio/thuy/Exercise4B.java

```diff
@@ -0,0 +1,40 @@
+package codeenginestudio.thuy;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+
+public class Exercise4B {
+    public static void main(String[] args) {
+
+        String[] listA = {"a", "b", "c", "d"};
+        String[] listB = {"d", "e"};
+        ArrayList<String> ouput = mergeArray(listA, listB, 1);
+        System.out.println("Ouput: " + ouput);
+
+    }
+
+    private static ArrayList<String> mergeArray(String[] listA, String[] listB, int index) {
+
+        ArrayList<String> aList = new ArrayList<String>(Arrays.asList(listA));
+        Collections.reverse(Arrays.asList(listB));
+
+        for (int i = 0; i < listB.length; i++) {
+            aList.add(index, listB[i]);
+        }
+        checkDuplicates(aList);
```

Comment:
Why duplicate here ?

Reply:
my mistake,, I check data,, but I forgot to change